tests/runtests.py

```diff
@@ -508,6 +510,11 @@ def paired_tests(paired_test, options, test_labels, parallel, start_at, start_af
         '-b', '--buffer', action='store_true',
         help='Discard output of passing tests.',
     )
+    parser.add_argument(
+        '-t', '--timing', action='store_true',
```

Comment:
Can we drop the `-t`, I'm not convinced it's needed and the short help just puts `-t` which is opaque (if you misspell the option).